test/system/700-play.bats

```diff
@@ -180,6 +180,47 @@ EOF
     run_podman 1 container exists $service_container
 }
 
+@test "podman play -- force-remove service-container" {
+    skip_if_remote "service containers only work locally"
+    # regression test for containers/podman/issues/16964
+
+    # Create the YAMl file
+    yaml_source="$PODMAN_TMPDIR/test.yaml"
+    cat >$yaml_source <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: test
+  name: test_pod
+spec:
+  restartPolicy: "Never"
+  containers:
+  - command:
+    - true
+    image: $IMAGE
+    name: test
+    resources: {}
+EOF
+
+    run_podman kube play --service-container --log-driver=journald $yaml_source
+
+    # The name of the service container is predictable: the first 12 characters
+    # of the hash of the YAML file followed by the "-service" suffix
+    yaml_sha=$(sha256sum $yaml_source)
+    service_container="${yaml_sha:0:12}-service"
+
+    # Remove the service container.
+    run_podman rm -f $service_container
+
```

Comment:
I still think this should not be allowed. How do you expect a user to know that this breaks the podman db? This is not documented.

Also we do not allow user to rm the infra container of a pod so why is removing the service container allowed then?

Reply:
> I still think this should not be allowed. How do you expect a user to know that this breaks the podman db? This is not documented.

It doesn't really break the database but I see what you mean.

> Also we do not allow user to rm the infra container of a pod so why is removing the service container allowed then?

That makes sense.  So a service-container should only be force removed which will also remove the associated pods and containers?  I like that.  I would still keep the current PR but remove the warning.